### DISCUSSION TITLE
连续预测报错，数量少预测也很慢。

### DISCUSSION THREAD

**Discussion by nissansz** [PaddlePaddle/PaddleOCR#12277] [Q&A] [answered]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：
- 版本号/Version：Paddle：2.3.2  PaddleOCR：2.7.5 问题相关组件/Related components：
- 运行指令/Command Code：
- 完整报错/Complete Error Message：

重置电脑后，安装完cuda, cudnn，推理报错了
>nvcc -V
nvcc: NVIDIA (R) Cuda compiler driver
Copyright (c) 2005-2020 NVIDIA Corporation
Built on Wed_Jul_22_19:09:35_Pacific_Daylight_Time_2020
Cuda compilation tools, release 11.0, V11.0.221
Build cuda_11.0_bu.relgpu_drvr445TC445_37.28845127_0

    ExternalError: CUDNN error(4), CUDNN_STATUS_INTERNAL_ERROR.
      [Hint: 'CUDNN_STATUS_INTERNAL_ERROR'.  An internal cuDNN operation failed.  ] (at ..\paddle\phi\kernels\gpu\rnn_kernel.cu.cc:88)
      [operator < rnn > error]

**Comment by UserWangZz** [comment]
可以检查一下CUDA版本与Paddle版本是否匹配

**Comment by nissansz** [comment]
paddlepaddle 2.3.2, cuda 10.2

**Comment by UserWangZz** [comment]
可以尝试paddle2.5.2版本

**Comment by nissansz** [comment, marked as answer]
尝试了，与aug不兼容，放弃
